[2024-01-02T20:34:02.022+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-10-15T00:00:00+00:00 [queued]>
[2024-01-02T20:34:02.030+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-10-15T00:00:00+00:00 [queued]>
[2024-01-02T20:34:02.031+0100] {taskinstance.py:1361} INFO - Starting attempt 3 of 6
[2024-01-02T20:34:02.043+0100] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-10-15 00:00:00+00:00
[2024-01-02T20:34:02.046+0100] {standard_task_runner.py:57} INFO - Started process 62355 to run task
[2024-01-02T20:34:02.053+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2022-10-15T00:00:00+00:00', '--job-id', '14267', '--raw', '--subdir', '/Users/user/Desktop/Learning/AIRFLOW-TUTORIAL/methode-1/dags/dag_with_postgres.py', '--cfg-path', '/var/folders/gn/cr4yj18d1tv34pkws9v_kq1r0000gn/T/tmpq_x99jgy']
[2024-01-02T20:34:02.057+0100] {standard_task_runner.py:85} INFO - Job 14267: Subtask insert_into_table
[2024-01-02T20:34:02.106+0100] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2022-10-15T00:00:00+00:00 [running]> on host macbook-pro-de-user.local
[2024-01-02T20:34:02.698+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='aida' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-10-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-15T00:00:00+00:00'
[2024-01-02T20:34:02.699+0100] {sql.py:274} INFO - Executing: 
            insert into dag_runs(dt, dag_id) values('2022-10-15','dag_with_postgres_operator_v03')
        
[2024-01-02T20:34:02.705+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T20:34:02.878+0100] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-01-02T20:34:02.908+0100] {sql.py:418} INFO - Running statement: 
            insert into dag_runs(dt, dag_id) values('2022-10-15','dag_with_postgres_operator_v03')
        , parameters: None
[2024-01-02T20:34:02.910+0100] {sql.py:427} INFO - Rows affected: 1
[2024-01-02T20:34:02.916+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20221015T000000, start_date=20240102T193402, end_date=20240102T193402
[2024-01-02T20:34:02.965+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-02T20:34:02.981+0100] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
